varats/varats/tools/driver_casestudy.py

```diff
@@ -579,13 +610,42 @@ def remove_error_result_files() -> None:
                     #  deleted files.
 
     def remove_result_files_by_regex() -> None:
-        pass
-        # TODO: Implement the removal of files specified by the user
-        #  https://github.com/se-passau/VaRA/issues/488
+        filter = args["filter"]
+        if filter == "":
```

Comment:
Empty strings are `False` in python.

```suggestion
        if not filter:
```